_posts/2020-07-27-climate-at-bcc.md

```diff
@@ -0,0 +1,28 @@
+---
+site: freiburg
+title: "Climate Science at BCC2020"
+supporters:
+ - galaxy-europe
+ - eosc
+---
+
+July has been a very busy month for [Galaxy Climate](https://climate.usegalaxy.eu/) ending with several sessions dedicated to Climate Science at [BCC2020](https://bcc2020.github.io/). It started on July 18-19 with two introductory training sessions (West and East) on “[Getting your hands on Climate data](https://training.galaxyproject.org/training-material/topics/climate/tutorials/climate-101/tutorial.html)” where attendees learned on accessing and analyzing climate data in Galaxy. One important aspect was to explain the difference between climate and weather data; show how to visualize climate data on a map with Galaxy and then how to create and share a simple workflow for framing a very simple adaptation case study.
+This event was supported by [Galaxy Europe](https://usegalaxy.eu/) and [TIaaS](https://galaxyproject.eu/tiaas) and benefited from the allocation of completely dedicated compute resources for the duration of the training.
+
+During the main BCC2020 conference [EOSC-Nordic](https://www.eosc-nordic.eu/) gave an update about Galaxy Climate and more importantly detailed its roadmap. The Galaxy Climate Science Workbench is a use case in the EOSC-Nordic project. At a policy level, the project will help to resolve issues related to cross border research collaboration using HPC and HPC-cloud based computing resources within EOSC. At a community level, the focus was put on fostering and advancing take-up through the development of blended training. Since GCC2019, EOSC-Nordic contributed to the development of 4 new Galaxy tutorials. For future developments, the approach is simple: rather than reinventing the wheel, EOSC-Nordic chose to base its roadmap on the [EOSC-Life tools roadmap] (https://github.com/eosc-life/tools-collaboratory-roadmap) and reuse as many components as possible for the Climate community. Strengthening collaboration with Galaxy Community and [EOSC-Life](https://www.eosc-life.eu/) is an integral part of its plan! 
```

Comment:
```suggestion
During the main BCC2020 conference [EOSC-Nordic](https://www.eosc-nordic.eu/) gave an update about Galaxy Climate and more importantly detailed its roadmap. The Galaxy Climate Science Workbench is a use case in the EOSC-Nordic project. At a policy level, the project will help to resolve issues related to cross border research collaboration using HPC and HPC-cloud based computing resources within EOSC. At a community level, the focus was put on fostering and advancing take-up through the development of blended training. Since GCC2019, EOSC-Nordic contributed to the development of 4 new Galaxy tutorials. For future developments, the approach is simple: rather than reinventing the wheel, EOSC-Nordic chose to base its roadmap on the [EOSC-Life tools roadmap](https://github.com/eosc-life/tools-collaboratory-roadmap) and reuse as many components as possible for the Climate community. Strengthening collaboration with Galaxy Community and [EOSC-Life](https://www.eosc-life.eu/) is an integral part of its plan! 
```